glossary.hbs

```diff
@@ -0,0 +1,72 @@
+{{!< default}}
+
+<main class="site__content article">
+    <div class="section background-alternate">
```

Comment:
Nebylo by vhodnější použít spíš `section` místo `div`?

Reply:
Nemyslím si. Z hlediska přístupnosti je pro orientaci podstatnější struktura nadpisů. H1 tu popisuje celý obsah `main`, takže odříznout ho od zbytku obsahu se `section` by nedávalo smysl. `section`s jsou pak použité v jednotlivých skupinách, ale opět platí, že to je spíš doplňková funkce, samotné nadpisy stačí.

(Zdroj: [Inclusive Components](http://book.inclusive-components.design/))